### ISSUE TITLE
Title should probably be above media in media module

### ISSUE THREAD

**Issue by surma** [GoogleChrome/devsummit#178]
![Chrome_Dev_Summit](https://user-images.githubusercontent.com/234957/62072157-f4b4a100-b235-11e9-8313-a5f26a5149b2.jpg)

Or the other way around: Media should not protrude top edge of text

**Comment by jakearchibald** [comment]
Up to you @argyleink. I don't mind it.

**Comment by argyleink** [comment]
surma, ugh, love these questions and point outs, thanks for taking the time to share your thoughts.

here's the reasoning and distinction between those 2 modules, and then a bug you helped me notice / something the design is poorly articulating.

### sessions module
this is a hub module summarizing the sessions collection. its following the pattern that whenever a home hub is representing a collection: we provide a link to the full list AND show the top 5 or a grouped/summed view. like, here's a taste of the best stuff, click one to see it, or click this link to all of them. this pattern is repeated across the homepage when a collection is shown. it's a ux pattern i really like 🤷‍♂ 

### about module
this module doesnt represent a summarized collection. it's a portable one off, represents no collections, so has no header link, needs to header to be above media or content, can live contextually inline.

### the bug!
i feel like your eye was drawn to the issue because in my design comp, the title and text in the about module are "almost" at the same top alignment as the sessions module block. so close in fact, you thought they should merge. what i intended in the about module, was left aligned content that's vertically centered with the media on the right. Why i call it a bug or bad design, is that the content is currently left aligned, but it's not vertically centered! it's off by 20-30px or so, and enough to not really even give the vertically centered vibe. 

tldr; collection modules need a header above the content so the link to it's full list is meaningfully located, where the about module should have an inline header to not disrupt the pattern the design is trying to instill. 

whatcha think!?

**Comment by argyleink** [comment]
note, the above "ugh" is not a groan of sorrow, it's a joyous victorious "ugh" after kicking some ass in battle

**Comment by surma** [comment]
Then our current design is correctly implemented 🎉 

Closing this.
